.github/workflows/stylelint.yml

```
@@ -0,0 +1,25 @@
+name: Code Quality - Stylelint
+
+on:
+  pull_request:
+    branches: [trunk]
+  push:
+    branches: [trunk,add/linting-github-actions]
+
+jobs:
+  stylelint:
+    name: stylelint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: install node v12
+      uses: actions/setup-node@v1
```

```suggestion
      uses: actions/setup-node@270253e841af726300e85d718a5f606959b2903c # v2.4.1
```

Upgrades v1 to the latest setup-node v2.4.1 release https://github.com/actions/setup-node/releases/tag/v2.4.1